packages/terra-functional-testing/src/webpack-server/webpack-server.js

```diff
@@ -23,9 +23,11 @@ class WebpackServer {
 
     // eslint-disable-next-line global-require, import/no-dynamic-require
     const config = require(webpackConfig);
+    const defaultLocale = process.env.LOCALE || locale;
+    const defaultTheme = process.env.THEME || theme;
```

Comment:
Could the env be applied before theme and locale are passed to config? Ideally we'd contain checking the env to one location. I think? 🤷‍♂️